### ISSUE TITLE
barcode_options not allowed?

### ISSUE THREAD

**Issue by LurkerAtTheGate** [usemarkup/BarcodeBundle#1]
it seems like your Configuration doesn't allow anything in barcode_options...
ex:

```
markup_barcode:
    definitions:
        invoice:
            type: itf14
            format: png
            barcode_options:   
               withBearerBars: true
```

It could be that I don't understand your intended Configuration yaml, but the above is what I would expect.  Changing the Configuration tree to
->arrayNode('barcode_options')
   ->prototype('scalar')->end()
->end()
allows the above to work (with as many options under barcode_options as is needed).

**Comment by shieldo** [comment]
I think you're going to want:

``` php
->prototype('variable')->end()
```

for that, but yes, you're dead right, thanks.

**Comment by shieldo** [comment]
@LurkerAtTheGate Thanks for the PR to fix this! I've made a `1.1.1` tag.
